packages/brokeneck-react/src/components/ErrorBoundary.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { screen, render } from '@testing-library/react'
+
+import ErrorBoundary from './ErrorBoundary'
+
+const ThrowError = ({ shouldThrow }) => {
+  if (shouldThrow) {
+    throw new Error('Custom error message')
+  } else {
+    return null
+  }
+}
+
+beforeEach(() => {
+  // eslint-disable-next-line no-console
+  console.error = jest.fn()
```

Comment:
why are you mocking this?

Reply:
This is being mocked because even when the test passes because the error is being handled the terminal logs the `console.error` message and clutters the test output. This avoids that unnecessary logging